src/jdk.jdi/windows/native/libdt_shmem/shmem_md.c

```diff
@@ -44,7 +44,7 @@ sysSharedMemCreate(const char *name, int length,
                    sys_shmem_t *mem, void **buffer)
 {
     void *mappedMemory;
-    HANDLE memHandle;
+ // HANDLE memHandle;
```

Comment:
I wasn't sure whether the global memHandle not being used was a bug, so I commented out the local one. I missed the line 88 one because it wasn't flagged. If it really isn't needed I'll remove that one instead

Reply:
I'm not sure what you mean by "that one". It's the static one that should be removed. The local variables always hide the static, and there seems to be no reason for the value of memHandle to survive outside of the local scope it is used in.

Reply:
Oh, I was referring to the static global one. In that case I'll remove it (Or rather, comment it so it isn't lost to time and someone can bring it back if need be)

Reply:
I would just remove the static. I don't see any reason for it to exist.